app.py

```diff
@@ -84,6 +97,21 @@ def send_new_channel_message(web_client: slack.WebClient, report_channel: str, n
     web_client.chat_postMessage(**message)
 
 
+def send_new_user_message(web_client: slack.WebClient, report_channel: str, new_user: str) -> None:
+    """Send a message to a channel about a new user event. Automatically pins the message.
+
+    :param web_client: The client to respond on
+    :param report_channel: The channel to send the message to
+    :param new_user: The new slack user object
+    """
+    new_user_message = NewUserMessage(report_channel, new_user)
+    message = new_user_message.get_message_payload()
+
+    response = web_client.chat_postMessage(**message)
+
+    # Pin this message to the channel
+    web_client.pins_add(channel=response['channel'], timestamp=response['ts'])
```

Comment:
How do we handle if pins are full?

Reply:
cry?

Reply:
We could check for a "too_many_pins" error, get all the pins in the channel, remove the last one, and retry, but lookoing at the docs, bots can't yet get the list of pins of a channel???

Reply:
Hmm.... For now maybe just send a message that tags @channel when it happens so we can manually handle? Not ideal but it's something

Reply:
I'll merge this now and we can just manually keep an eye on the number of pins, removing the old ones. I'll keep checking the docs to see if they add support for app bots to check in on pins.

Reply:
https://github.com/OSU-CS/automod/issues/34